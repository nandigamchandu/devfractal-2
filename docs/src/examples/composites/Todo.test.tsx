import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Todo } from './Todo'

test('Todo Example', () => {
  const component: ReactTestRenderer = renderer.create(<Todo />)
  expect(component.toJSON()).toMatchSnapshot()
})
