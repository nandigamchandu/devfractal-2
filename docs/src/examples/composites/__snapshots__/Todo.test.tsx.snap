// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Example 1`] = `
<section
  className="section"
>
  <table
    className="table"
  >
    <thead
      className=""
    >
      <tr
        className="tr"
      >
        <td
          className=""
        >
          Id
        </td>
        <td
          className=""
        >
          Title
        </td>
        <td
          className=""
        >
          Description
        </td>
        <td
          className=""
        >
          Time
        </td>
        <td
          className=""
        >
          Done
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className="tr"
      >
        <td
          className=""
        >
          1
        </td>
        <td
          className=""
        >
          bring milk
        </td>
        <td
          className=""
        >
          I love tea!
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <input
              className="input"
              readOnly={true}
              type="date"
              value="2019-04-17"
            />
          </div>
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <label
              className="checkbox"
            >
              <input
                checked={true}
                className="checkbox"
                readOnly={true}
                type="checkbox"
              />
            </label>
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className=""
        >
          2
        </td>
        <td
          className=""
        >
          go for a walk
        </td>
        <td
          className=""
        >
          need to keep fit
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <input
              className="input"
              readOnly={true}
              type="date"
              value="2019-04-17"
            />
          </div>
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <label
              className="checkbox"
            >
              <input
                checked={false}
                className="checkbox"
                readOnly={true}
                type="checkbox"
              />
            </label>
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className=""
        >
          3
        </td>
        <td
          className=""
        >
          read news
        </td>
        <td
          className=""
        >
          I really lack general knowledge
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <input
              className="input"
              readOnly={true}
              type="date"
              value="2019-04-17"
            />
          </div>
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <label
              className="checkbox"
            >
              <input
                checked={false}
                className="checkbox"
                readOnly={true}
                type="checkbox"
              />
            </label>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;
